azure/scope/managedcontrolplane.go

```diff
@@ -655,3 +657,23 @@ func (s *ManagedControlPlaneScope) TagsSpecs() []azure.TagsSpec {
 		},
 	}
 }
+
+// AvailabilityStatusResource refers to the AzureManagedControlPlane.
+func (s *ManagedControlPlaneScope) AvailabilityStatusResource() conditions.Setter {
+	return s.ControlPlane
+}
+
+// AvailabilityStatusResourceURI constructs the ID of the underlying AKS resource.
+func (s *ManagedControlPlaneScope) AvailabilityStatusResourceURI() string {
+	return azure.ManagedClusterID(s.SubscriptionID(), s.ResourceGroup(), s.ControlPlane.Name)
+}
+
+// AvailabilityStatusFilter ignores the health metrics connection error that
+// occurs on startup for every AKS cluster.
+func (s *ManagedControlPlaneScope) AvailabilityStatusFilter(cond *clusterv1.Condition) *clusterv1.Condition {
+	if time.Since(s.ControlPlane.CreationTimestamp.Time) < 1*time.Hour &&
```

Comment:
nit, let's change this `1*time.Hour` to a const like `resourceHealthEventualConsistencyTolerance` or something